docker/docker_start.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/bash -x
```

Comment:
There seem to be a lot of duplicated code. What is the main difference start.sh and docker_start.sh?

Reply:
`docker_start.sh` is the old starter script with some additional environment setup. Binaries are looked up from a different place.

`start.sh` is the "new" code. The previous file can be deleted, it has only been preserved so that the current Dockerfile does not break in any way.

Reply:
OK. I think you can remove the old script as well as dockerfile. We don't need to keep both

Reply:
ok